pydicom/pixel_data_handlers/util.py

```diff
@@ -244,7 +244,7 @@ def apply_modality_lut(arr: "np.ndarray", ds: "Dataset") -> "np.ndarray":
     * DICOM Standard, Part 4, :dcm:`Annex N.2.1.1
       <part04/sect_N.2.html#sect_N.2.1.1>`
     """
-    if 'ModalityLUTSequence' in ds:
+    if len(ds.get("ModalityLUTSequence", [])):
```

Comment:
Could skip that `len`, same below - should be sufficient to have
```py
if ds.get("ModalityLUTSequence"):
```

Reply:
But if `ds.ModalityLUTSequence = []` then the check will pass but cause issue #1554, won't it?

Reply:
No, it won't pass - an empty array is a falsy value. And I think pylint would also not like that length check (though we don't run it here).

Reply:
Ah, yes, of course